06_puppy/sirigithub/stores_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storesSuite struct {
+	suite.Suite
+	store     Storer
+	newStorer func() Storer
+	puppy1    Puppy
+}
+
+func (suite *storesSuite) SetupTest() {
+	suite.store = suite.newStorer()
+	suite.puppy1 = Puppy{Breed: "cavalier", Color: "White", Value: "$1700"}
+	suite.store.CreatePuppy(&suite.puppy1)
+}
+func (suite *storesSuite) TestCreatePuppySuccesful() {
```

Comment:
nit: please put a blank line between functions (as you have in the other files). It makes it easier to scan the code.

Reply:
@camh-anz included blank lines in between functions.